implementation/domain/logic/jbb-system-impl/src/main/java/org/jbb/system/impl/session/model/SessionImpl.java

```diff
@@ -0,0 +1,106 @@
+package org.jbb.system.impl.session.model;
+
+import org.hibernate.validator.constraints.NotBlank;
+import org.hibernate.validator.constraints.NotEmpty;
+import org.jbb.system.api.model.session.UserSession;
+
+import java.time.Duration;
+import java.time.LocalDateTime;
+import java.time.temporal.ChronoUnit;
+
+import javax.validation.constraints.NotNull;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Setter;
+
+@Setter
+@Getter
+@Builder
+@AllArgsConstructor
+public class SessionImpl implements UserSession {
+
+    @NotNull
+    @NotEmpty
+    @NotBlank
+    private String id;
+
+    @NotNull
+    private LocalDateTime creationTime;
+
+    @NotNull
+    private LocalDateTime lastAccessedTime;
+
+    @NotNull
+    private Duration usedTime;
```

Comment:
Redundant field. Used time can be calculated as (last accessed time) minus (creation time)